source/scenario/globalScenarioTriggers.vwf.yaml

```diff
@@ -274,7 +274,7 @@ properties:
       triggerCondition:
       - onBlocklyBlock:
         - rover3
-        - rover_turn
+        - 'Turn: Left'
```

Comment:
Also, we should still make sounds while the rover is turning - just not when it's executing blocks *other* than move or turn.  And we will likely have custom sounds for other blocks (like the spray-can sound that gets turned on when drawing triangles, perhaps sounds for sensors, ...)